src/main/java/net/sf/jabref/logic/util/io/FileUtil.java

```diff
@@ -111,46 +110,36 @@
     /**
      * Copies a file.
      *
-     * @param source         File Source file
-     * @param dest           File Destination file
-     * @param deleteIfExists boolean Determines whether the copy goes on even if the file
-     *                       exists.
-     * @return boolean Whether the copy succeeded, or was stopped due to the
-     * file already existing.
+     * @param pathToSourceFile Path Source file
+     * @param pathToDestinationFile Path Destination file
+     * @param replaceExisting boolean Determines whether the copy goes on even if the file exists.
+     * @return boolean Whether the copy succeeded, or was stopped due to the file already existing.
      * @throws IOException
      */
-    public static boolean copyFile(File source, File dest, boolean deleteIfExists) throws IOException {
+    public static boolean copyFile(Path pathToSourceFile, Path pathToDestinationFile, boolean replaceExisting) throws IOException {
         // Check if the file already exists.
-        if (dest.exists() && !deleteIfExists) {
+        if (!Files.exists(pathToSourceFile)) {
             return false;
```

Comment:
I think it would be nice if you would log here, why the file wasn't copied. So here because the path to the source file not exists.